templates/controllers_deployment.yml

```diff
@@ -47,6 +50,27 @@ spec:
         volumeMounts:
           - name: uaa-client-secret
             mountPath: /config/uaa_client_secret
+      - name: backup-metadata-generator
+        image: #@ data.values.images.backup_metadata
+        imagePullPolicy: Always
+        command: [ "/cnb/process/wait" ]
+        env:
+        - name: CF_API_HOST
+          value: #@ "http://capi.{}.svc.cluster.local".format(data.values.system_namespace)
+        - name: CF_CLIENT
+          value: cf_api_controllers
+        - name: CF_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: password
+              name: #@ data.values.uaa.clients.cf_api_controllers.secret_name
```

Comment:
Will update this once this is merged:
https://github.com/cloudfoundry/cf-for-k8s/pull/555